employee_self_service/mobile/v1/feed.py

```diff
@@ -69,7 +69,13 @@ def get_ess_post(post_name):
             "answer",
         )
         post_details["total_vote"] = len(post_details.get("ess_post_poll_log"))
-    del post_details["ess_post_poll_log"]
+        
+    if frappe.session.user != post_details.get('user'):
+        del post_details["ess_post_poll_log"]

```

Comment:
Remove **ess_post_poll_log** information if poll is not created by the data requested user.